Eatery Blue/UI/Profile/StayLoggedInSheet.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  StayLoggedInSheet.swift
+//  Eatery Blue
+//
+//  Created by Jayson Hahn on 11/7/23.
+//
+
+import Foundation
+import UIKit
+
+class StayLoggedInSheet: SheetViewController {
+    
+    private let descriptionLabel = UILabel()
+    private var delegate : AttemptLogin?
```

Comment:
```suggestion
    private weak var delegate: AttemptLogin?
```

Reply:
I'm getting this erro could you clarify what this means google is confusing. "'weak' must not be applied to non-class-bound 'any AttemptLogin'; consider adding a protocol conformance that has a class bound"